js/msp/msp_helperfunctions.js

```diff
@@ -0,0 +1,1813 @@
+'use strict';
+MSP.listen(MSP_process_data);
+function MSP_process_data (dataHandler) {
```

Comment:
Going from methods defined on an object (`MSP`) to globally defined functions seems to be a huge step backwards. If anything, we should define an `MspHelper` define these as methods on it.